### PR TITLE
Config fixes

### PR DESCRIPTION
**Pull request by dknochen**
- several fixes for private cellstore deployment configurations
- allow using proxies

depends on https://github.com/28msec/28/pull/63

### CODE REVIEW

**Comment by dknochen** [comment]
@wcandillon this is now ready for review. Thank you.

File: tasks/tests.js [hunk 1]
```diff
@@ -22,6 +23,9 @@ gulp.task('tests:e2e', ['tests:update-webdriver'], function() {
   }
   var specs = Config.paths.e2eSpecs;
   $.util.log('specs: ' + specs);
+  if(!_.isArray(specs) || specs.length === 0){
```
Comment (wcandillon) [inline tasks/tests.js]: @dknochen Why this change?

**Comment by wcandillon** [comment]
@dknochen I am not sure why you authorize empty spec lists, other than that it's good.

File: tasks/tests.js (hunk 1 above)
Comment (dknochen) [inline tasks/tests.js]: Otherwise, you have to create an e2e test before creating and testing a new cellstore config.

**Comment by dknochen** [comment]
Otherwise, you have to create an e2e test before creating and testing a new cellstore config. Its just to remove the obstacles. Alternatively, we could add a default e2e test that is run for any deployment (e.g. the auth-scenario.js). What do you think?

**Comment by wcandillon** [comment]
@dknochen I like the idea to have the e2e auth test running for all scenarios.

**Comment by dknochen** [comment]
@wcandillon I also like it :)
